client.go

```diff
@@ -225,6 +215,50 @@ func (c *Client) UpdateFeature(feature *Feature) error {
 	return nil
 }
 
+func (c *Client) getProjectID(feature *Feature) (int64, error) {
+	if feature.ProjectID != nil {
+		return *feature.ProjectID, nil
+	}
+	project, err := c.GetProject(feature.ProjectUUID)
+
+	if err != nil {
+		return 0, err
+	}
+	return project.ID, nil
+}
+
+func (c *Client) manageFeatureOwners(feature *Feature, ownerIDs []int64, endpoint string) error {
+	projectID, err := c.getProjectID(feature)
+	if err != nil {
+		return err
+	}
+	url := fmt.Sprintf("%s/projects/%d/features/%d/%s/", c.baseURL, projectID, *feature.ID, endpoint)
+	body := struct {
+		UserIDs []int64 `json:"user_ids"`
+	}{
+		UserIDs: ownerIDs,
+	}
+	resp, err := c.client.R().SetBody(body).Post(url)
+
+	if err != nil {
+		return err
+	}
+
+	if !resp.IsSuccess() {
+		return fmt.Errorf("flagsmithapi: Error %s owners from feature: %s", endpoint, resp)
```

Comment:
This error is a bit unpleasant. If the endpoint is `add-owners` then we get: "flagsmithapi: Error add-owners from feature: feature_x" which doesn't make much sense. I guess we have a couple of options: 

 1. Be less user-friendly with the error message and just have something like: `"Error calling %s endpoint: %s", endpoint, resp`
 2. Separate these out into two separate methods. We could still make the middle part shared between the two but the error handling should be done by each individual function.

I tend to like the second option here.